sdk/log/exporter.go

```diff
@@ -201,6 +200,10 @@ func newBufferExporter(exporter Exporter, size int) *bufferExporter {
 	}
 }
 
+func (e *bufferExporter) IsQueueFull() bool {
+	return len(e.input) == cap(e.input)
```

Comment:
I think this can have a data race as access to `e.input` is not synchronized.
Shouldn't we `IsQueue` full be implemented in`queue` struct? E.g.

```go
func (q *queue) Full() bool {
	q.Lock()
	defer q.Unlock()
	return q.len == q.cap
}
```